### PR TITLE
Update Windows Installation instruction in Setting-Up-a-Metasploit-Development-Environment.md

### PR DESCRIPTION
**Pull request by soroshsabz**
ITNOA

Add PowerShell based installation instructions

### CODE REVIEW

**Comment by soroshsabz** [comment]
@h00die Do you can review this PR?

thanks

File: docs/metasploit-framework.wiki/dev/Setting-Up-a-Metasploit-Development-Environment.md [hunk 1]
```diff
@@ -30,10 +30,29 @@ sudo apt update && sudo apt install -y git autoconf build-essential libpcap-dev
 
 ### Windows
 
-If you are running a Windows machine
+If you are running a Windows 10 or above machine
```
Comment (h00die) [inline docs/metasploit-framework.wiki/dev/Setting-Up-a-Metasploit-Development-Environment.md:33]: `#### Windows 10 or above`

File: docs/metasploit-framework.wiki/dev/Setting-Up-a-Metasploit-Development-Environment.md [hunk 2]
```diff
@@ -30,10 +30,29 @@ sudo apt update && sudo apt install -y git autoconf build-essential libpcap-dev
...
+```
+
+Install a version of PostgreSQL:
+
+```
+Install-Module -Name Microsoft.WinGet.Client
+Install-WinGetPackage -id PostgreSQL.PostgreSQL.17
+```
+
+If you are running a old Windows machine
```
Comment (h00die) [inline docs/metasploit-framework.wiki/dev/Setting-Up-a-Metasploit-Development-Environment.md:52]: `#### Pre-Windows 10`

**Review by h00die — DISMISSED** [review]

**Comment by soroshsabz** [comment]
@h00die I resolved all your comments

**Review by jheysel-r7 — COMMENTED** [review]
Thanks for the update @soroshsabz! I followed the updated instructions and was able to install metasploit on Windows 10 without any issues 👍 
Screenshot 2024-11-29 at 11 44 11 AM

**Review by jheysel-r7 — APPROVED** [review]

**Pull request merged**